flepimop/gempyor_pkg/src/gempyor/calibrate.py

```diff
@@ -1,3 +1,11 @@
+"""
+Facilitates the calibration of the model using the `emcee` MCMC sampler. 
+Provides CLI options for users to specify simulation paramters. 
```

Comment:
```suggestion
Facilitates the calibration of the model using the `emcee` MCMC sampler. 

Provides CLI options for users to specify simulation parameters. 
```
Split into header and description (assuming that's the intent?) and typo.